quipucords/fingerprinter/task.py

```diff
@@ -268,7 +277,8 @@ def _process_details_report(self, manager_interrupt, details_report):
             self.scan_task.log_message(status_message, log_level=logging.ERROR)
             deployment_report.status = DeploymentsReport.STATUS_FAILED
             status = ScanTask.FAILED
-        deployment_report.cached_json = json.dumps(final_fingerprint_list)
+        deployment_report.cached_fingerprints = json.dumps(
```

Comment:
not a blocker, (and i know this wasn't introduced in your pr) but it would be good to change this variable to `deployments_report` at some point)

Reply:
On second thought, lets definitely do this in a separate pr

Reply:
:trollface: